public/js/cita.json

```diff
@@ -5008,5 +4868,305 @@
     "lng": -2.5934541,
     "id": "94e94f50-1d30-49f0-8959-b2a799ba3423",
     "booking_location_id": "b74377da-d06b-48d0-9d74-cc362bbb53b3"
+  },
+  {
+    "title": "Poyton Civic Hall",
```

Comment:
I would suggest a title of "Poyton" and add the fact it is at the civic hall to the address.